modules/gapi/include/opencv2/gapi/own/cvdefs.hpp

```diff
@@ -137,6 +137,15 @@ enum InterpolationFlags{
     INTER_LINEAR_EXACT   = 5,
     INTER_MAX            = 7,
 };
+namespace gapi { namespace own {
+class Rect;
+class Size;
+class Point;
+} // namespace own
+} // namespace gapi
+using Rect  = gapi::own::Rect;
+using Size  = gapi::own::Size;
+using Point = gapi::own::Point;
 } // namespace cv
```

Comment:
I assume this file should include our own versions of Rect/Size/Point? Or there will be a cyclic dependency?

Reply:
@aDanPin @akhakim Added `#include <.../own/types.hpp>` for `cv::gapi::own:: Rect Size Point`  in `cvdefs.hpp`.

Removed `#include <.../own/types.hpp>` from files which have `#include<.../opencv_includes.hpp>`(cvdefs.hpp in STANDALONE mode)

Reply:
Fixed.